src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,97 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static final String VALID_LOGIN = "Username";
+    private static final String ONE_MORE_LOGIN = "Username2";
+    private static final String VALID_PASSWORD = "Password";
+    private static final String UNVALID_LOGIN = "12345";
+    private static final String UNVALID_PASSWORD = "12345";
+    private static final int VALID_AGE = 18;
+    private static RegistrationService registrationService = new RegistrationServiceImpl();
+    private static User validUser = new User();
+    private static User existUser = validUser;
+
+    @BeforeEach
+    void beforeEach() {
+        validUser.setAge(VALID_AGE);
+        validUser.setLogin(VALID_LOGIN);
+        validUser.setPassword(VALID_PASSWORD);
+        validUser.setId((long) 12);
+    }
```

Comment:
It is bad practice to use the same data for all tests. Better to create your own for each test. This improves readability and makes test cases more consistent